### PR TITLE
Objects Automated Test - LPS-140558 | master

### PR DESCRIPTION
**Pull request by PauloVAS**

### CODE REVIEW

**Comment by PauloVAS** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/tests/ObjectPortlet.testcase [hunk 1]
```diff
@@ -167,12 +167,63 @@ definition {
...
+
+		Picklist.clickSaveButton();
+
+		Picklist.gotoPicklists();
+
+		Picklist.gotoPicklistView(picklistName = "Custom Picklist");
+
+		Picklist.addPicklistItemViaUI(picklistItemName = "Item Test 2");
+
+		Picklist.clickSaveButton();
```
Comment (rodrigocunhaa) [inline modules/apps/object/object-test/src/testFunctional/tests/tests/ObjectPortlet.testcase:190]: This should be done via API. There are already created macros under `Picklist.macro`.

File: modules/apps/object/object-test/src/testFunctional/tests/tests/ObjectPortlet.testcase [hunk 2]
```diff
@@ -167,12 +167,63 @@ definition {
...
+		ObjectAdmin.addObjectFieldViaUI(
+			fieldLabel = "Field Picklist",
+			fieldPicklist = "Custom Picklist",
+			fieldType = "Picklist");
+
+		ObjectAdmin.goToDetailsTab();
+
+		CreateObject.selectPanelCategoryKey(panelCategoryKey = "Control Panel > Object");
+
+		CreateObject.saveObject();
```
Comment (rodrigocunhaa) [inline modules/apps/object/object-test/src/testFunctional/tests/tests/ObjectPortlet.testcase:213]: This should be done via API too.

File: modules/apps/object/object-test/src/testFunctional/tests/tests/ObjectPortlet.testcase [hunk 3]
```diff
@@ -167,12 +167,63 @@ definition {
...
+
+		ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject");
+
+		ApplicationsMenu.gotoPortlet(
+			category = "Object",
+			panel = "Control Panel",
+			portlet = "Custom Objects");
```
Comment (rodrigocunhaa) [inline modules/apps/object/object-test/src/testFunctional/tests/tests/ObjectPortlet.testcase:220]: Use the `ObjectAdmin.goToCustomObject(objectName = "CustomObject");` instead.

**Review by rodrigocunhaa — DISMISSED** [review]
Hi @PauloVAS ,

Please, make the changes that I requested.

Let me know if you have any question.

**Comment by rodrigocunhaa** [comment]
ci:forward

**Pull request closed without merging**